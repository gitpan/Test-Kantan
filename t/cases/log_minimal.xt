use strict;
use Test::Kantan;
use Test::MockTime qw(set_absolute_time);
set_absolute_time(1394852288);
use Log::Minimal;

local $Log::Minimal::PRINT = sub {
    my ( $time, $type, $message, $trace,$raw_message) = @_;
    local $Test::Kantan::Level = $Test::Kantan::Level + 3;
    Test::Kantan::diag("$time [$type] $message at $trace", 1024);
};
infof("Hey");

ok { 0 };
done_testing;
