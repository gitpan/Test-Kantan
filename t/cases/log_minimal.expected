




  (Diagnostic message):


      ♨
      2014-03-15T11:58:08 [INFO] Hey at t/cases/log_minimal.xt line 12
         at t/cases/log_minimal.xt line 12

      ✖
      ok { 0 };
         at t/cases/log_minimal.xt line 14


not ok
1..1
